features/facebookImport/src/model/caching.js

```diff
@@ -0,0 +1,80 @@
+import EntityDeserializer from "./entities/utils/entity-serialisation";
+
+const namespace = "http://polypoly.coop/schema/fbImport/";
+
+export default class ExportDataCache {
+    constructor(pod, fileId) {
+        this._pod = pod;
+        this._fileId = fileId;
+    }
+
+    _createAccessQuery(dataFactory) {
+        return {
+            subject: dataFactory.namedNode(`${this._fileId}/cache`),
+            predicate: dataFactory.namedNode(`${namespace}importerCache`),
+        };
+    }
+
+    async _selectDataQuad(dataFactory, polyIn) {
+        return await polyIn.select(this._createAccessQuery(dataFactory));
+    }
+
+    _createDataQuad(dataFactory, rawData) {
+        return dataFactory.quad(
+            dataFactory.namedNode(`${this._fileId}/cache`),
```

Comment:
If you're using this repeatedly, maybe you should have a method for it so that you have a single place to change instead of several ones.